generators/lua/lists.js

```diff
@@ -363,3 +363,20 @@ Blockly.Lua['lists_split'] = function(block) {
   var code = functionName + '(' + input + ', ' + delimiter + ')';
   return [code, Blockly.Lua.ORDER_HIGH];
 };
+
+Blockly.Lua['lists_reverse'] = function(block) {
+  // Block for reversing a list.
+  var list = Blockly.Lua.valueToCode(block, 'LIST',
+      Blockly.Lua.ORDER_NONE) || '{}';
+  var functionName = Blockly.Lua.provideFunction_(
+      'list_reverse',
+      ['function ' + Blockly.Lua.FUNCTION_NAME_PLACEHOLDER_ + '(l)',
+       '  local t = {}',
```

Comment:
Sure. I just copied the variable names that are used in all the other Lua functions in this file.